client/webserver/site/src/js/login.ts

```diff
@@ -13,6 +13,16 @@ export default class LoginPage extends BasePage {
     Doc.show(this.form)
     this.loginForm = new LoginForm(this.form, () => { this.loggedIn() })
     this.loginForm.focus()
+
+    app().registerNoteFeeder({
+      login: (note: LoginNote) => { this.handleLoginNote(note) }
+    })
```

Comment:
It took me a while to figure out why I wasn't getting the notes when using Login from `/register`.
I think we need to have this handler registered from `LoginForm` instead of `LoginPage`.

```diff
diff --git a/client/webserver/site/src/js/forms.ts b/client/webserver/site/src/js/forms.ts
index 5f738eb5a..cea6f1a2b 100644
--- a/client/webserver/site/src/js/forms.ts
+++ b/client/webserver/site/src/js/forms.ts
@@ -22,7 +22,8 @@ import {
   WalletStateNote,
   WalletInfo,
   Token,
-  WalletCreationNote
+  WalletCreationNote,
+  CoreNote
 } from './registry'
 import { XYRangeHandler } from './opts'
 
@@ -1573,12 +1574,21 @@ export class LoginForm {
     this.headerTxt = page.header.textContent || ''
 
     bind(form, page.submit, () => { this.submit() })
+
+    app().registerNoteFeeder({
+      login: (note: CoreNote) => { this.handleLoginNote(note) }
+    })
   }
 
   focus () {
     this.page.pw.focus()
   }
 
+  handleLoginNote (n: CoreNote) {
+    const loginMsg = Doc.idel(this.form, 'loaderMsg')
+    if (loginMsg) loginMsg.textContent = n.subject
+  }
+
   async submit () {
     const page = this.page
     Doc.hide(page.errMsg)
```

Any down sides to that?

Reply:
> It took me a while to figure out why I wasn't getting the notes when using Login from /register.

Oops, thanks.

> Any down sides to that?

Nah, all good. The login note handler will be deregistered when a new page loads.